files/soma.js/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "soma.js"
+github = "https://github.com/somajs/somajs"
+homepage = "http://somajs.github.com/somajs/"
+description = "soma.js is a scalable javascript framework created to help developers to write loosely-coupled applications to increase scalability and maintainability."
+mainfile = "build/soma-v2.1.1.min.js"
```

Comment:
This would be a great solution, since it's not a bleeding edge library like other frameworks, such as Angular, Aurelia, etc. :+1: